### PR TITLE
misc: style change for responsiveness

### LINKED ISSUE #25664: Debug M1.1 - Correct responsiveness of DebugNewRelevantRunBar
**Current behavior**

When the breakpoint is really narrow, the DebugNewRelevantRunBar does not wrap and looses a lot of information.

**Desired behavior**
The DebugNewRelevantRunBar should wrap when the breakpoint is narrow instead of staying on one line and being truncated.

### PR DESCRIPTION
**Pull request by marktnoonan**
- Closes #25664 

### Additional details

This modifies some CSS to get a better experience in cases where the content of this component is wider than a single row.

### Steps to test

Mainly I would look at the percy snapshots. The situations where this can wrap in the app are pretty rare and would involve long commit messages or high zoom levels since the debug page has a minimum width.

To test this in the live app, you can open Cypress to the Debug page when a run is in progress and modify the DOM of the "next run" display to include unusually long text.

### How has the user experience changed?

Old on left, new on right: 

Screen Shot 2023-02-02 at 11 28 51 AM

### PR Tasks

- [na] Have tests been added/updated? - percy
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by ZachJW34 — APPROVED** [review]
I pushed a changelog entry! Changes look good

**Review by lmiller1990 — APPROVED** [review]

**Review by warrensplayer — COMMENTED** [review]
image

Not a big deal, but curious why there is more spacing between the 2nd and 3rd lines vs the 1st and 2nd.

**Comment by marktnoonan** [comment]
@warrensplayer it has to do with the separator, mostly - the vertical space between each list item is 8px, but each one is a different size internally.

I really enjoy making stuff like this perfect, but didn't think it was worth spending time on it in this case that's so hard to hit in the app.

File: cli/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
  <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.6.0
+
+_Released 02/14/2023 (PENDING)_
+
+**Misc:**
+
+- Improved the responsiveness of the Debug Page pending run UI. Addresses [#25664](https://github.com/cypress-io/cypress/issues/25664).
```
Comment (mike-plummer) [inline cli/CHANGELOG.md]: Wordsmithing nitpick: "Improved the layout of the Debug Page on smaller viewports when there is a pending run."

Maybe it's just me, but I interpret the phrase "improved responsiveness" as "performance improvement"

**Review by mike-plummer — APPROVED** [review]
One suggestion for the changelog, but otherwise looks good 👍

File: cli/CHANGELOG.md (hunk 1 above)
Comment (marktnoonan) [inline cli/CHANGELOG.md]:
```suggestion
- Improved the layout of the Debug Page on smaller viewports when there is a pending run. Addresses [#25664](https://github.com/cypress-io/cypress/issues/25664).
```

**Pull request merged**